lib/versioning/poetry/poetry.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { api as semver } from '../../lib/versioning/poetry';
+import { api as semver } from '.';
```

Comment:
I think we should rename these types of files to `index.spec.ts` now that they're within `lib`. Please check all versioning, manager and datasource ones

Reply:
okay will change the names

Reply:
Done with this too.